### PR TITLE
Change information_schema to uppercase

### PR DESCRIPTION
**Pull request by WilliamBZA**
This change should also include a change to the Sql Server used in the test environment to be case sensitive

* [x] Change TC server to be case sensitive

Fixes https://github.com/Particular/NServiceBus.Persistence.Sql/issues/347

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Case sensitivity for sql server is now tested during the test phase

File: src/TestHelper/MsSqlConnectionBuilder.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static void Setup(string tenantId)
             using (var conn = MsSqlConnectionBuilder.Build())
             {
                 conn.Open();
-                conn.ExecuteCommand($"if not exists (select * from sysdatabases where name = '{dbName}') create database {dbName};");
+                conn.ExecuteCommand($"if not exists (select * from sysdatabases where name = '{dbName}') create database {dbName}; ALTER DATABASE [{dbName}] COLLATE SQL_Latin1_General_CP1_CS_AS;");
```
Comment (tmasternak) [inline src/TestHelper/MsSqlConnectionBuilder.cs]: What happens on the build agent when the db already exists?

Comment (WilliamBZA) [reply src/TestHelper/MsSqlConnectionBuilder.cs]: The command just executes.

Comment (WilliamBZA) [reply src/TestHelper/MsSqlConnectionBuilder.cs]: I take that back, it only executes if the schema is already case sensitive...

**Comment by WilliamBZA** [comment]
Closing this as I want to take a different tack on this one.

**Pull request closed without merging**